conjur/cli.py

```diff
@@ -231,6 +292,34 @@ def run_client_action(resource, args):
             # A successful exit is required to prevent the initialization of
             # the Client because the init command does not require the Client
             return
+        elif resource == 'login':
+            credential_data = CredentialData(None, args.name, None)
+            credentials = CredentialsFromFile()
+            login_logic = LoginLogic(credentials)
+            login_controller = LoginController(ssl_verify=args.ssl_verify,
+                                               password=args.password,
+                                               credential_data=credential_data,
+                                               login_logic=login_logic)
+            login_controller.load()
+            return
+        elif resource == 'logout':
+            credentials = CredentialsFromFile()
+            logout_logic = LogoutLogic(credentials)
+
+            logout_controller = LogoutController(ssl_verify=None,
+                                                 logout_logic=logout_logic)
```

Comment:
I think this i redundant, just `logout_logic is enough.
```suggestion
                                                 logout_logic)
```

Reply:
These are keyword arguments <key>=<value>. I cannot do it for one (logout_logic_ and not for the other (ssl_verify) above. I prefer to leave it to be explicit